packages/typedoc-plugins/lib/tag-observable/index.js

```diff
@@ -22,7 +22,11 @@ module.exports = {
 
 function onEventEnd( context ) {
 	// Get all resolved reflections that could potentially have the `@observable` tag.
-	const reflections = context.project.getReflectionsByKind( ReflectionKind.Property );
+	const reflections = [
+		...context.project.getReflectionsByKind( ReflectionKind.Property ),
+		...context.project.getReflectionsByKind( ReflectionKind.GetSignature ),
+		...context.project.getReflectionsByKind( ReflectionKind.SetSignature )
+	];
```

Comment:
Minor improvement: instead of traversing 3 times the whole project, you can get all requested reflections at once.

```suggestion
	const kinds = ReflectionKind.Property | ReflectionKind.GetSignature | ReflectionKind.SetSignature;
	const reflections = context.project.getReflectionsByKind( kinds );
```